src/format-emoji-blot.js

```diff
@@ -1,7 +1,9 @@
-import Parchment from 'parchment';
+import Quill from 'quill';
 import emojiMap from "./emoji-map";
 
-class EmojiBlot extends Parchment.Embed {
+const Embed = Quill.import('blots/embed');
+
+class EmojiBlot extends Embed {
```

Comment:
@mcorrand here previous logic to build this component because I had some error when I try to build it with 'Parchment'.